services/app-web/src/pages/StateSubmission/RateDetails/RateDetailsSchema.ts

```diff
@@ -10,9 +10,15 @@ import {
 Yup.addMethod(Yup.date, 'validateDateFormat', validateDateFormat)
 
 const SingleRateCertSchema = (_activeFeatureFlags: FeatureFlagSettings) =>
+    // Yup.object().shape({
+    //     linkRateSelect: Yup.string().defined('wowow'),
+    // })
     Yup.object().when('.ratePreviouslySubmitted', {
         // this first when is skipping all validations for a previously submitted rate.
         is: 'YES',
+        then: Yup.object().shape({
+            linkRateSelect: Yup.string().defined('You must select a rate certification'),
```

Comment:
Hmm good point I see that now too!

Reply:
This issue seems like a blocker since its our basic form validation functionality and with multi rate its super hard to see where errors are without the error summary work.

Other things mentioned I agree with for sure and want to discuss --  but don't think they should block us merging a fix to bugs that prevent key form actions. Sounds right @JasonLin0991 ?

Reply:
Agree! Here is the ticket for link rate error summary and links https://jiraent.cms.gov/browse/MCR-4105